php/Terminus/Models/Environment.php

```diff
@@ -775,21 +763,4 @@ private function getSettings($setting = null) {
 
   }
 
-  /**
-   * Changes the environment's settings
-   *
-   * @param array $settings Key/value pairs to set in the environment settings
-   * @return bool
-   */
-  private function updateSetting(array $settings = []) {
```

Comment:
@TeslaDethray This function doesn't seem to be used anymore in this codebase - I would remove it, but I'm not sure if that impacts plugins or has any other undesirable effects.